### PR TITLE
React Native Release 4.4.0

### PR DESCRIPTION
**Pull request by AnatolyPristensky**
* [x] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Bumped the versions
Updated TestApp to use latest SDKs
Updated the CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,19 +1,46 @@
...
+* **[Fix]** Remove `android.support.test.InstrumentationRegistry` string that caused an error when checking applications on availability of android support libraries.
+* **[Feature]** Add `AppCenter.setCountryCode(string)` API to set the country code manually.
+
 ### App Center Analytics
 
 #### Android/iOS
 
  * **[Feature]** Add `Analytics.enableManualSessionTracker` and `Analytics.startSession` APIs for tracking session manually.
 
+* **[Feature]** Increase the interval between sending logs from 3 to 6 seconds for the backend load optimization.
```
Comment (AnastasiaKubova) [inline CHANGELOG.md:28]: pls remove an [empty line](https://github.com/microsoft/appcenter-sdk-react-native/pull/945/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL14)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,19 +1,46 @@
...
+
+#### Android
+
+* **[Feature]** Add `AppCenter.setLogger` API to set custom logger.
```
Comment (AnastasiaKubova) [inline CHANGELOG.md:32]: I'm not sure that this feature is related to React Native

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,19 +1,46 @@
...
+
 ### App Center Crashes
 
+#### iOS
+
+* **[Feature]** Add `(NSString *)description` method to convert `MSACErrorReport` to a string and additional useful information about sending error.
```
Comment (AnastasiaKubova) [inline CHANGELOG.md:38]: I'm not sure that this feature is related to React Native

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,19 +1,46 @@
...
+* **[Feature]** Save a crash report from the Xamarin.Mac platform.
```
Comment (AnastasiaKubova) [inline CHANGELOG.md:39]: and this

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,19 +1,46 @@
...
+* **[Fix]** Fix build failure on Xcode 13, because of warning `Сompletion handler is never used`. Only observable when SDK is integrated as source code. Continuation of the previous fix that fixed the issue on the beta version.
```
Comment (AnastasiaKubova) [inline CHANGELOG.md:40]: and this

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,19 +1,46 @@
...
 
 ### App Center
 
+#### Android/iOS
+
 * **[Breaking change]** Remove `AppCenter.setCustomProperties` API.
 
+#### iOS
+
+* **[Fix]** Fix `Undefined symbol: OBJC_CLASS$_CTTelephonyNetworkInfo` error for Mac Catalyst platform when integrating the SDK via Swift Package Manager with Swift 5.5 and higher.
```
Comment (AnastasiaKubova) [inline CHANGELOG.md:13]: Not related to React Native

**Review by AnastasiaKubova — APPROVED** [review]

File: AppCenterReactNativeShared/Products/local.podspec [hunk 7]
```diff
@@ -3,7 +3,7 @@
 
 Pod::Spec.new do |s|
   s.name              = 'AppCenterReactNativeShared'
-  s.version           = '4.3.0'
+  s.version           = '4.4.1'
```
Comment (DmitriyKirakosyan) [inline AppCenterReactNativeShared/Products/local.podspec:6]: Why 4.4.1? I think it should be 4.4.0.

**Review by DmitriyKirakosyan — APPROVED** [review]

**Pull request merged**